### PR TITLE
fixes a broken projects link on the docs homepage

### PR DESCRIPTION
**Pull request by osis**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

The PROJECTS.md link on https://open-cap-table-coalition.github.io/Open-Cap-Format-OCF/ is a broken link. This PR points it to the correct page.

### CODE REVIEW

**Review by JSv4 — APPROVED** [review]
Thx

File: docs/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ OCF is a JSON-based format, wherein individual objects required to describe a co
 | --------------------------------------------------------: | ------------------------------------------------------------ |
 | [Primary Website](https://www.opencaptablecoalition.com/) | Sharable landing site for OCF and the Coalition              |
 |           [Medium Page](https://medium.com/@opencaptable) | Past announcements related to OCF and the Coalition          |
-|                          [PROJECTS.md](/docs/PROJECTS.md) | Collection of public projects in the OCF ecosystem listed in |
+|             [PROJECTS.md](/Open-Cap-Format-OCF/PROJECTS/) | Collection of public projects in the OCF ecosystem listed in |
```
Comment (JSv4) [inline docs/README.md:15]:
```suggestion
| [PROJECTS.md](https://open-cap-table-coalition.github.io/Open-Cap-Format-OCF/PROJECTS/) | Collection of public projects in the OCF ecosystem listed in |
```

**Review by JSv4 — CHANGES_REQUESTED** [review]
Actually, on second thought, can we direct this to the projects page on the documentation website? Generally we don't want to direct people to the docs folder in the repo.

**Comment by osis** [comment]
LGTM

**Comment by osis** [comment]
On a second look, the original link I put in does indeed go to the documents page and not the file. It doesn't include the domain so it makes the path less brittle. Unless you saw something that I didn't @JSv4?

**Pull request closed without merging**